platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationService.java

```diff
@@ -75,4 +75,29 @@
    *     <code>null</code>
    */
   MigrationReport doImport(Path exportDirectory, Consumer<MigrationMessage> consumer);
+
+  /**
+   * Decrypts an exported file from the specified path.
+   *
+   * @param exportDirectory path to decrypt configurations from
+   * @return a migration report for the decrypt operation
+   * @throws IllegalArgumentException if <code>exportDirectory</code> is <code>null</code>
```

Comment:
❓ Shouldn't this also include invalid `exportDirectory` values, not just `null`? See my comment on the `newZipOutputStreamFor` method.

Reply:
No as those would be treated as migration errors and would fail the migration and be reported as such. Here we only validate the argument passed in which should not be null.